readme.md

```diff
@@ -163,48 +169,53 @@ How are we defining columns in `create_table`?
 * We then use methods like `.string` or `.integer` to represent the data types of the columns we are creating.
 * These methods take in a symbol argument (e.g., `:photo_url`) that represents the name of the column.  
 
-> For a full list of SQL data types, click [here](http://www.w3schools.com/sql/sql_datatypes_general.asp). For more on using the `change` method, click [here](http://edgeguides.rubyonrails.org/active_record_migrations.html#using-the-change-method)  
+> For a full list of SQL data types, click [here](http://www.w3schools.com/sql/sql_datatypes_general.asp).
+>
+> For more on using the `change` method, click [here](http://edgeguides.rubyonrails.org/active_record_migrations.html#using-the-change-method)  
 
 You can create the `artists` table and run this migration by entering the following into the terminal...
 
 ```bash
 $ rails db:migrate
 ```
 
-After running this, a file called `db/schema.rb` is generated. What are it's contents?
+After running this, a file called `db/schema.rb` is generated. What are its contents?
 
-> You should NEVER have to update the `schema.rb`. Running `rails db:migrate` will update the schema for you using migration files.  
+> You should **NEVER** have to update the `schema.rb`. Running `rails db:migrate` will update the schema for you using migration files.  
 
 ## You Do: View `db/schema.rb` (5 minutes / 0:40)
 
-**[1 minute]** Read through this new file.
+> 2 minutes exercise. 3 minutes review.
+
+Read through this new file.
 
 ## You Do: Use Rails Console (5 minutes / 0:45)
 
-**[2 minutes]** Enter the Rails Console and create two Artists.
-* You can do this using the Active Record methods you learned in earlier classes.  
+> 2 minutes exercise. 3 minutes review.
 
-> Remember, you can enter the Rails Console by typing `rails c` into the Terminal.  
+Enter the Rails console -- `rails c` -- and create two Artists using Active Record methods.
 
 ## You Do: Songs Migration (10 minutes / 0:55)
 
-**[5 minutes]** Create a migration file for a new `songs` table. Songs should have columns for `title` `album` and `preview_url`, all of which are strings.  
+> 5 minutes exercise. 5 minutes review.
+
+Create a migration file for a new `songs` table. Songs should have columns for `title` `album` and `preview_url`, all of which are strings.  
 
-To associate one model with another in a migration file, you can include one of the following methods in the `create_table` method. They all do the same thing assuming you have set up your associations properly in your models...  
+To associate one model with another in a migration file, you can include one -- **and only one!** -- of the following methods in the `create_table` method. They all do the same thing assuming you have set up your associations properly in your models...  
 * `t.belongs_to :artist`
 * `t.references :artist`
 * `t.integer :artist_id`
```

Comment:
Be prepared to talk about how this is distinct from the associations set up in our models.